src/__tests__/radium-test.js

```diff
@@ -276,6 +276,40 @@ describe('Radium blackbox tests', () => {
     expect(nav.style.color).to.equal('blue');
   });
 
+  it('passes snapshot to the componentDidUpdate of the component, Issue #985', () => {
+    const SNAPSHOT = 'SNAPSHOT';
+
+    @Radium
+    class SnapshotComp extends Component {
+      state = {
+        incrementToTriggerLifecycles: 0
+      };
+
+      componentDidUpdate(props, state, snapshot) {
```

Comment:
Should we also assert this was actually called just to cover our bases?

Reply:
Updated, and refactored a bit to be simpler.